README.md

```diff
@@ -177,3 +177,12 @@ ADD_CUSTOM_COMMAND (OUTPUT ${GENERATED_STUB}
 
 The usage of the `$GENERATED_STUB` files will trigger the execution of the code
 generator.
+
+## License and copyright
+This README file is Copyright &copy; 2018 Luxoft Sweden AB
+SPDX-License-Identifier: CC-BY-SA-4.0
+[!License: CC BY-SA 4.0](https://img.shields.io/badge/LICENSE-CC--BY--SA--4.0-lightgrey.svg)](https://creativecommons.org/licenses/by-nc/4.0/)
+
+
+The python code is Copyright &copy; 2008-2011 Red Hat, Inc.
```

Comment:
Please add a line for Luxoft (looking at the commit history, it looks like we started working on it in 2014).